src/ActionList/Group.tsx

```diff
@@ -58,32 +79,64 @@ export const Group: React.FC<React.PropsWithChildren<ActionListGroupProps>> = ({
       }}
       {...props}
     >
-      {title && <Header title={title} variant={variant} auxiliaryText={auxiliaryText} labelId={labelId} />}
+      {(slots.groupHeading || title) && (
```

Comment:
Since we're dealing with types where the false-y value may be unexpected (like empty string), would it help to use a ternary here so the false-y branch reliably outputs `null`? Reference for the broader idea here: https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

It may not actually be that helpful here so totally understand if not, just noticed the values themselves aren't booleans and had this come up as a thought 👍

Reply:
I love that, thank you for sharing! I think I am so caught up in the convenience of using `&&` sometimes I forget about javascript quirks. Yes, empty string would mess this up. I'll update the code 💖

Reply:
committed [here](https://github.com/primer/react/pull/3900/commits/b929c0046bffc86cc94625993afc2c398cf49516) 🙂